src/main/kotlin/org/jitsi/nlj/AudioLevelListener.kt

```diff
@@ -17,5 +17,5 @@
 package org.jitsi.nlj
 
 interface AudioLevelListener {
-    fun onLevelReceived(sourceSsrc: Long, level: Long)
```

Comment:
Document somewhere that the return value means discard?

Reply:
done